tests/testthat/test-compiler-ghpages.R

```diff
@@ -22,9 +23,41 @@ test_that("error for viz with no source", {
   }, "plot.list does not contain option named source, which is required by animint2pages")
 })
 
-test_that("animint2pages() returns owner/repo string", {
-  viz_owner_repo <- animint2pages(viz, github_repo = "animint2pages_test_repo")
-  expect_is(viz_owner_repo, "character")
+## This test requires a fine-grained PAT
+## - go to https://github.com/settings/personal-access-tokens/new
+## - Resource owner: animint-test
+## - Repository access: All repositories
+## - Repository permissions: Administration and Contents: read and write.
+## - Generate token
+## - copy token and paste into PAT_GITHUB on https://github.com/animint/animint2/settings/secrets/actions
```

Comment:
I saw that it has already been set, so I don't need to set the PAT again, correct? 
Will the PAT expire?

Reply:
I set the PAT to expire in 1 year (that is the max on github)
If you try to run the test locally, it should work, as long as your github token has the right permissions.
Since I added you to the animint-test org as member (can create new repositories), you should be able to run the test code locally with your regular github credentials (no special token required).

Reply:
actually I just tried that myself and I got
```r
> gh::gh("DELETE /repos/animint-test/animint2pages_test_repo")
Error in `gh::gh()`:
! GitHub API error (403): Must have admin rights to Repository.
ℹ Read more at <https://docs.github.com/rest/repos/repos#delete-a-repository>
Run `rlang::last_trace()` to see where the error occurred.
```
but I am owner of animint-test org???

Reply:
I had to change base permissions of animint-test org from Read to Admin
![image](https://github.com/user-attachments/assets/3a3e2714-da3a-48ee-94eb-98d93cafe042)
https://github.com/organizations/animint-test/settings/member_privileges?enable_tip=#base-permissions

Reply:
https://github.com/animint-test/animint2pages_test_repo/settings/access says base role is Admin??
![image](https://github.com/user-attachments/assets/a3323215-a915-475b-9c47-129518a0eea3)

Reply:
so I got it to work for local testing by doing the following.
first I used `gitcreds::gitcreds_delete()` to remove my previous git crendential/token, which I believe only has repo read/write permission (not delete).
then I go to https://github.com/settings/tokens/new and check `repo` and `delete_repo` then generate token, as shown below.
![image](https://github.com/user-attachments/assets/78c4e695-f7b8-4a86-bafe-799949f68ec7)
Then I open up git bash on windows, type git push, then a window pops up, click token, paste the token, confirm.
Then running the tests locally should work.

Reply:
I added some LOCAL TESTING docs in the comments next to this test, does that make sense to you?

Reply:
Yes, thank you! I will try on my local tonight.

Reply:
I just tested it locally and got it to work! 
It seems `gitcreds::gitcreds_delete()` wasn't deleting my old token, so I used `credentials::set_github_pat("my-pat")` to reset it.

Reply:
great thanks, I moved the docs to the wiki https://github.com/animint/animint2/wiki/Testing#installing-a-github-token-for-use-locally and I mentioned your `credentials::set_github_pat("my-pat")` trick there.
hopefully this technique will make it easier to implement tests in #138